modules/apps/account/account-service/service.xml

```diff
@@ -48,6 +48,36 @@
 		<reference entity="Resource" package-path="com.liferay.portal" />
 		<reference entity="User" package-path="com.liferay.portal" />
 	</entity>
+	<entity local-service="true" name="AccountEntryOrganizationRel">
+
+		<!-- PK fields -->
+
+		<column name="accountEntryOrganizationRelId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+
+		<!-- Other fields -->
+
+		<column name="accountEntryId" type="long" />
+		<column name="organizationId" type="long" />
+
+		<!-- Finder methods -->
+
+		<finder name="AEI" return-type="Collection">
+			<finder-column name="accountEntryId" />
+		</finder>
+		<finder name="AEI_O" return-type="AccountEntryOrganizationRel">
+			<finder-column name="accountEntryId" />
+			<finder-column name="organizationId" />
+		</finder>
```

Comment:
Since there is no conflicting column name that starts with the letter `a`, we can safely change this finder name to be `A_O`. The primary key column starts with `a`, but it is identified in the finder as `primaryKey`, so there shouldn't be a conflict.